modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -54,6 +54,10 @@ def _read(cls, filepath_or_buffer, **kwargs):
         # Ensures that the file is a string file path. Otherwise, default to pandas.
         filepath_or_buffer = cls.get_path_or_buffer(filepath_or_buffer)
         if isinstance(filepath_or_buffer, str):
+            if "*" not in filepath_or_buffer:
+                warnings.warn(
+                    f"'*' symbol not found in filename: '{filepath_or_buffer}'"
```

Comment:
The error message should not only state the error condition, but _why_ it's probably an error; for now it's just stating "hey, no star in name", and a user could be "so what?".

Warning should explain what's going on or give a reference where to find more.

Reply:
Updated

Reply:
@vnlitvinov ready for review